src/chrome/content/rules/tricolor.tv.xml

```diff
@@ -0,0 +1,33 @@
+<!--
+tricolor.tv ²
+www.tricolor.tv ²
+blog.tricolor.tv ¹
+dealers.tricolor.tv ²
+kino.tricolor.tv ²
+old.tricolor.tv ²
+organizations.tricolor.tv ²
+shop.tricolor.tv ³
+goldcard.tricolor.tv ²
+nochnoy.tricolor.tv ²
+help.tricolor.tv ²
+sales.tricolor.tv ³
+magazine.tricolor.tv ²
+spb.tricolor.tv ²
+www1.tricolor.tv ²
+radio.tricolor.tv ²
+perm.tricolor.tv ²
+krasnodar.tricolor.tv ²
+stavropol.tricolor.tv ²
```

Comment:
`tricolor.tv`, `m.tricolor.tv` and `www.tricolor.tv` are working for me now, double check them.